frontend/management/src/components/TableBody.tsx

```diff
@@ -0,0 +1,31 @@
+import {ApplicantInterface, Column, UserInterface} from "../utils/types.tsx";
+import "../styles/table.css"
+import TableRowUsers from "./TableRowUsers.tsx";
+import TableRowApplicants from "./TableRowApplicants.tsx";
+
+function TableBody({ columns, tableData } : Props) {
+  return (
+    <tbody className="table__body">
+    {tableData.map((data : UserInterface | ApplicantInterface)  => {
+      if ("role" in data) { // Users
```

Comment:
Maybe we can make an explicit attribute in the object, which would signalise, whether it is an applicant or a user?

Reply:
I agree, maybe there is another way to see if it has a certain type, but I couldn't get it to work.

Reply:
Let's pass this piece of info from backend, for now this solution is fine